### PR TITLE
Add Volumes To Remote Copy Groups for HTTP

### PR DESCRIPTION
**Pull request by nirupaman**

### CODE REVIEW

**Comment by nirupaman** [comment]
@farhan7500 Please review

File: test/HPE3ParMockServer_flask.py [hunk 1]
```diff
@@ -821,6 +821,24 @@ def getPort(portPos):
...
+       memname = volstr[0].strip()
+       if memname != 'hostname':
+          ret={}
+          volume_name = volstr[1].strip()
+          for vlun in vluns['members']:
+             if vlun['volumeName'] == volume_name:
+                ret['members']=[vlun]
+
+          resp = flask.make_response(json.dumps(ret), 200)
+          return resp
```
Comment (farhan7500) [inline test/HPE3ParMockServer_flask.py]: Please set resp value in if else and return resp only at the end

if 'query' in flask.request.args:
    ....
    ....
    
else:
   ....
   ....
return resp

**Review by farhan7500 — CHANGES_REQUESTED** [review]
Please check if we can cover more by generating the code coverage of the remote copy module.

File: test/HPE3ParMockServer_flask.py (hunk 1 above)
Comment (nirupaman) [reply test/HPE3ParMockServer_flask.py]: have change the code to return resp at the end

File: test/HPE3ParMockServer_flask.py [hunk 2]
```diff
@@ -821,7 +821,25 @@ def getPort(portPos):
...
+       if memname != 'hostname':
+          ret={}
+          volume_name = volstr[1].strip()
+          for vlun in vluns['members']:
+             if vlun['volumeName'] == volume_name:
+                ret['members']=[vlun]
+
+          resp = flask.make_response(json.dumps(ret), 200)
+    else:
+	resp = flask.make_response(json.dumps(vluns), 200)
```
Comment (farhan7500) [inline test/HPE3ParMockServer_flask.py]: Why are there 2 space separation in else? We need to be consistent. Is this really working?

**Comment by nirupaman** [comment]
@farhan7500 If changes i made is correct or something more need to done in function

**Review by farhan7500 — CHANGES_REQUESTED** [review]

**Comment by farhan7500** [comment]
Also, when I run the unit tests for coverage, there are indentation issue. Please fix all indentation issues so we can run the code coverage

Command: sudo tox -e cover test/test_HPE3ParClient_volume.py

 File "<frozen importlib._bootstrap>", line 693, in _load
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 661, in exec_module
  File "<frozen importlib._bootstrap_external>", line 767, in get_code
  File "<frozen importlib._bootstrap_external>", line 727, in source_to_code
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "/home/nirun/hpe3par_python_sdk/test/test_HPE3ParClient_volume.py", line 724
    self.cl.FLASH_CACHE_ENABLED = 1
                                  ^
TabError: inconsistent use of tabs and spaces in indentation

**Comment by nirupaman** [comment]
@farhan7500    tabs and spaces removed

**Review by farhan7500 — APPROVED** [review]

**Pull request merged**